packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -146,7 +154,7 @@ export abstract class Credentials {
    * If no password is provided, one will be generated and stored in SecretsManager.
    */
   public static fromUsername(username: string, options: CredentialsFromUsernameOptions = {}): Credentials {
-    return { username, password: options.password, encryptionKey: options.encryptionKey };
+    return { username, password: options.password, encryptionKey: options.encryptionKey, excludeCharacters: options.excludeCharacters };
```

Comment:
nit: prefer if we have a property per line. this is readable but sometimes i need to do a double-take

Reply:
Done!